samples/api-client/accesstoken_example/accesstoken_test.py

```diff
@@ -0,0 +1,280 @@
+# Copyright 2021 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import base64
+import os
+import sys
+import time
+import uuid
+
+import requests as req
+
+# Add command receiver for bootstrapping device registry / device for testing
+
+sys.path.append(os.path.join(os.path.dirname(__file__), "..", "manager"))  # noqa
+
+import manager  # noqa
+
+import accesstoken  # noqa
+
+cloud_region = "us-central1"
+device_id_template = "test-device-{}"
+rsa_cert_path = "resources/rsa_cert.pem"
+rsa_private_path = "resources/rsa_private.pem"  # Must match rsa_cert
+device_topic_id = "test-device-events-{}".format(uuid.uuid4())
+gcs_bucket_name = "test-bucket-name-{}".format(uuid.uuid4())
+project_id = os.environ["GOOGLE_CLOUD_PROJECT"]
+service_account_json = os.environ["GOOGLE_APPLICATION_CREDENTIALS"]
+device_pubsub_topic = "projects/{}/topics/{}".format(project_id, device_topic_id)
+test_topic_id = "test-pubsub-{}".format(uuid.uuid4())
+
+# This format is used in the `clean_up_registries()` below.
+registry_id = "test-registry-{}-{}".format(uuid.uuid4().hex, int(time.time()))
+
+# Generate gcp access token, use gcp access token to create pubsub
+
+
+def test_generate_gcp_jwt_token_pubsub():
```

Comment:
same comments for moving tests to the accesstoken.py as "feature" with the same region tags. see https://github.com/googleapis/nodejs-iot/pull/502